airgun/views/host_new.py

```diff
@@ -63,35 +77,145 @@ class Overview(Tab):
-            ROOT = '(.//article[contains(@class, "pf-c-card")])[1]'
+            ROOT = './/article[.//div[text()="Details"]]'
             details = HostDetailsCard()
 
         @View.nested
         class HostStatusCard(Card):
-            ROOT = '(//article[contains(@class, "pf-c-card")])[2]'
+            ROOT = './/article[.//span[text()="Host status"]]'
             status = Text('.//h4[contains(@data-ouia-component-id, "OUIA-Generated-Title")]')
 
-            status_success = Text('.//span[@class="status-success"]')
-            status_warning = Text('.//span[@class="status-warning"]')
-            status_error = Text('.//span[@class="status-error"]')
-            status_disabled = Text('.//span[@class="disabled"]')
+            status_success = Text('.//span[span[@class="status-success"]]')
+            status_warning = Text('.//span[span[@class="status-warning"]]')
+            status_error = Text('.//span[span[@class="status-error"]]')
+            status_disabled = Text('.//span[span[@class="disabled"]]')
+
+        @View.nested
+        class InstallableErrataCard(Card):
+            ROOT = './/article[.//div[text()="Installable errata"]]'
+
+            security_advisory = Text('.//a[contains(@href, "type=security")]')
+            bug_fixes = Text('.//a[contains(@href, "type=bugfix")]')
+            enhancements = Text('.//a[contains(@href, "type=enhancement")]')
+
+        @View.nested
+        class TotalRisksCard(Card):
+            ROOT = './/article[.//div[text()="Total risks"]]'
+
+            low = Text('.//*[@id="legend-labels-0"]/*')
+            moderate = Text('.//*[@id="legend-labels-1"]/*')
+            important = Text('.//*[@id="legend-labels-2"]/*')
+            critical = Text('.//*[@id="legend-labels-3"]/*')
 
         @View.nested
         class RecentJobsCard(Card):
-            ROOT = '(//article[contains(@class, "pf-c-card")])[5]'
+            ROOT = './/article[.//div[text()="Recent jobs"]]'
             is_table_loaded = './/ul[@aria-label="recent-jobs-table"]'
 
     @View.nested
     class Content(Tab):
-        pass
+        ROOT = './/div'
 
-    @View.nested
-    class Traces(Tab):
-        enable_traces = Button('OUIA-Generated-Button-primary-1')
+        @View.nested
+        class Packages(Tab):
+            ROOT = './/div[@id="packages-tab"]'
+
+            select_all = Checkbox(locator='.//div[@id="selection-checkbox"]/div/label')
+            searchbar = TextInput(locator='.//input[contains(@class, "pf-m-search")]')
+            status_filter = Dropdown(locator='.//div[@aria-label="select Status container"]/div')
+            upgrade = Button('Upgrade')
+            dropdown = Dropdown(locator='.//div[button[@aria-label="bulk_actions"]]')
+
+            table = PatternflyTable(
+                component_id="host-packages-table",
+                column_widgets={
+                    0: Checkbox(locator='.//input[@type="checkbox"]'),
+                    'Package': Text('./parent::td'),
```

Comment:
Are these clickable cells?

Reply:
They could be if the package is installed from synced repo on satellite